movieapi/cinema/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    duration = models.IntegerField()
+    objects = models.Manager()
```

Comment:
I think it's unnecessary.
![image](https://user-images.githubusercontent.com/104218550/178724256-1610723a-df83-42ee-9d86-3e2667bbb693.png)

Reply:
Fixed